releng/release-notes-next/generate_buildrequires.bug

```diff
@@ -0,0 +1,19 @@
+When Mock completes the installation of all the requirements generated
+by `%generate_buildrequries`, it calls `rpmbuild -ba` to perform a final build
+of the package.
+
+During the final build, `%generate_buildrequries` runs again in order to
+generate a list of `BuildRequires` to be added to the built SRPM metadata.
+An arbitrary `%generate_buildrequries` section may generate different
+requirements that may not have been installed.
+
+Previously, the `rpmbuild -ba` call used the `--nodeps` option,
+hence it was possible to successfully build a package with
+unsatisfiable BuildRequires in the built SRPM metadata.
+
+When a bootstrap chroot is used, the `--nodeps` option is no longer used
+in the final `rpmbuild -ba` call.
```

Comment:
Can you use `[no longer used][PR#1249]` instead to link this PR in release notes, or link `[issue#1246]` somehow?